recommendation/microsoft.owin.json

```diff
@@ -136,7 +136,7 @@
               "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
             }
           ],
-          "Description": "Remove the base class of OwinMiddleware and replace the method modifiers for the invoke method to only public so as to remove the override modifier.",
+          "Description": "Remove the base class for owin middleware, remove the override method modifier for the invoke method, add a new gloable xpression for RequestDelegate _next, initialize it in the constructor and remove the constructor intializer base(next).",
```

Comment:
nits: 
* Change `owin middleware` to `OwinMiddleware` to match the style used in rules below
* Correct typo in `gloable xpression`

Reply:
Good catch thank you 👍